packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -74,6 +74,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   DragGestureRecognizer({
     super.debugOwner,
     this.dragStartBehavior = DragStartBehavior.start,
+    this.multitouchDragStrategy = MultitouchDragStrategy.trackLatestActivePointer,
```

Comment:
If we pursue this, we will likely want to maintain the current behavior.

Reply:
I prefer to switch the default behavior to the same as Android, because I think this experience is better, even on the ios platform. What do you think?
If the developer really wants the previous behavior, it can be achieved by overriding ScrollBehavior.

Reply:
```My first thought is, how would we clarify for developers the difference between this+DragGestureRecognizer and MultiDragGestureRecognizer```

Good point! I was confused at the time, and then I went to study `MultiDragGestureRecognizer`.
They do have some confusion in terms of names and are actually completely different things. 
`multitouchDragStrategy ` means to use multiple fingers to complete one dragging task. 
`MultiDragGestureRecognizer` is to distribute different finger events to different `MultiDragPointerState` instances to achieve the purpose of dragging multiple objects at the same time(see `Draggable` widget).

Reply:
Or we can change the name to `DragMultitouchStrategy` ?

Reply:
I'm also leaning towards maintaining the current behavior (at least in `DragGestureRecognizer`). Some users might be depending on the current behavior in tests, and with `DragGestureRecognizer` being used frequently and having been around for so long then maybe we shouldn't break this expectation.

Reply:
WRT to naming I like the current name but can see why it may be confusing given `MultiDragGestureRecognizer`, as a second choice I also like `dragMultitouchStrategy`.

Reply:
> I'm also leaning towards maintaining the current behavior (at least in `DragGestureRecognizer`). Some users might be depending on the current behavior in tests, and with `DragGestureRecognizer` being used frequently and having been around for so long then maybe we shouldn't break this expectation.

Make sense, I have changed  `DragGestureRecognizer` default to current behavior.

Also, I change the `ScrollBehavior.multitouchDragStrategy` to the new behavior for all platforms, it will cause all `Scrollable` to change to new behavior.

Reply:
There are a couple of folks that are out of office this week that I think would want to weigh in here. I will follow up on Tuesday 10/24 once I can get this change in front of them. :)

Reply:
Ok! I think we are ready to fix this! I chatted with @goderbauer 👍, and @Hixie 👍 , and I think @dkwingsmt also wanted to weigh in here.
I originally thought preserving the old behavior as the default was the right way to go in case it was a breaking change but I was wrong. Can you update this to have the default be the correct behavior? I can check to see if anything breaks, and if it does we can go from there with migration guidance and communicating the change more widely.
The way you have fashioned this is a pretty painless migration. Well done! 💯

Reply:
@dkwingsmt Hi, Could you take a look at this? : )

Reply:
Yes, I agree that we should provide the "correct" behavior out of the box, since dragging is such a fundamental feature. The current behavior should be considered a bug that we fix with this PR, while preserving an option for users to go back.

However, I have two more suggestions (sorry for suggesting them so late):
* We should default to the iOS-style strategy on iOS. Again, this is such a fundamental feature that we really should provide native-like behavior out of the box.
* We might be better off with a `InheritedWidget` that configures the entire app in one go instead of having to provide this argument for every widget, especially since many of the dragging widgets are buried in the depth of other widgets.

Reply:
> We might be better off with a InheritedWidget that configures the entire app in one go instead of having to provide this argument for every widget, especially since many of the dragging widgets are buried in the depth of other widgets.

I think that is the goal of using the ScrollBehavior here @dkwingsmt. It can be set for the entire app, and is inherited so all scrollables will use it. This change does not require every widget to be configured that way.

Reply:
Ah I see, I misunderstood the code. The current design makes sense!